app/helpers/application_helper/toolbar/cloud_subnets_center.rb

```diff
@@ -11,6 +11,7 @@ class ApplicationHelper::Toolbar::CloudSubnetsCenter < ApplicationHelper::Toolba
           'pficon pficon-add-circle-o fa-lg',
           t = N_('Add a new Cloud Subnet'),
           t,
+          :klass => ApplicationHelper::Button::CloudSubnetNew
```

Comment:
That will check against all providers while in a list view? Looking at the source for `GenericFeatureButtonWithDisable` suggests that only works on an individual record.